src/JavaScriptSDK.Interfaces/ITelemetryItem.ts

```diff
@@ -24,7 +24,7 @@ export interface ITelemetryItem {
     /**
      * Identifier of the resource that uniquely identifies which resource data is sent to
      */
-    ikey?: string;
+    iKey?: string;
```

Comment:
was this missed before u just published all the packages or just checking this in?

Reply:
just checking this in. Tests on other packages would fail otherwise